style.css

```
@@ -0,0 +1,3 @@
+h1 {
```

- [ ] [REQUIRED] You are using an element selector to style the heading, I kindly suggest using a class selector instead to follow the project requirements. You can add to class to the heading and use that class here to style the heading.
```suggestion
h1 {
```